docs/how-to-guides/advanced-user-guides/enable-user-retention.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Enabling User Retention
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/enable-user-retention.md"/>
+</head>
+
+In Rancher v2.8.5 and later (and v2.7.14 and later in the v2.7.x line), you can enable user retention to automatically disable or delete inactive user accounts after a configurable time period.
+
+The user retention feature is off by default.
+
+## Enabling User Retention with Kubectl
+
+To enable user retention, use kubectl to set `user-retention-cron`. Then, set `delete-inactive-user-after`, `disable-inactive-user-after`, or a combination of both:
+
+```
+kubectl edit setting user-retention-cron 
+kubectl edit setting disable-inactive-user-after
```

Comment:
The `### Required User Retention Settings` section mentions these expect certain values, but that's not indicated here.

Reply:
Originally, this was a placeholder for a working example, but the values were removed in https://github.com/rancher/rancher-docs/pull/1343/commits/2f4049e995c8d3ef24a3381282887d40ac774bc1 without clarifying. It might be better to state "use `kubectl edit setting` to ..." instead of providing the snippet